recipes/parallelproj/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 826a83e51db6459004d17c030c21fb32e7cb5205a45c28c8f50f87f235321196
+
+build:
+  number: {{ build_number }}
+  string: "h{{ PKG_HASH }}_{{ build_number }}_cpu"   # [cuda_compiler_version == "None"]
+  string: "h{{ PKG_HASH }}_{{ build_number }}_cuda"  # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - cmake >=3.23
+    - ninja
+  host:
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]
+test:
+  commands:
+    - test -f $PREFIX/lib/libparallelproj_c$SHLIB_EXT  # [unix]
+    - if not exist %LIBRARY_LIB%\parallelproj_c.lib exit 1  # [win]
```

Comment:
Typically win libraries have both .lib and .dll.

Reply:
On Windows the .dll gets installed in ```...Library/bin``` instead of  ```.../Library/lib```, what can I use instead of ```%LIBRARY_LIB%``` that gets expanded correctly?

Reply:
Yes. Check for both. `%LIBRARY_BIN%`

https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html

Reply:
done